dataclients/kubernetes/clusterclient.go

```diff
@@ -37,6 +37,8 @@ const (
 	serviceAccountRootCAKey    = "ca.crt"
 )
 
+const routeGroupsNotInstalledMessage = "RouteGroups CRD is not installed in the cluster."
```

Comment:
please point to user documentation https://opensource.zalando.com/skipper/kubernetes/routegroups/

Reply:
ok